spec/pg/decoder_spec.cr

```diff
@@ -1,4 +1,5 @@
 require "../spec_helper"
+require "uuid"
```

Comment:
also this wouldn't be necessary if changed to generate_series

Reply:
Done, thanks!